src/lorentz_boost/boost_parameter/boost_axis/convert.jl

```diff
@@ -0,0 +1,16 @@
+
+function Base.convert(
+    ::Type{B}, param::S
+) where {T<:Real,B<:AbstractAxisBoostParameter{T},S<:Real}
+    return B(T(param))
+end
+function Base.convert(
+    ::Type{B1}, d::B2
+) where {T<:Real,B1<:AbstractAxisBoostParameter{T},B2<:AbstractAxisBoostParameter}
+    return B1(T(d.param))
+end
+function Base.convert(
+    ::Type{B1}, d::B2
+) where {T<:Real,B1<:AbstractAxisBoostParameter{T},B2<:AbstractAxisBoostParameter{T}}
+    return d
+end
```

Comment:
Indeed, this is a no-op, but it is necessary to avoid calling the convert function from above (line 7). To make this more clear, I just simplified the signature in b230698